adcm_client/objects.py

```diff
@@ -937,8 +937,16 @@ def _upload(self, bundle_stream) -> Bundle:
         return self.bundle(bundle_id=data['id'])
 
     @allure_step('Upload bundle from "{1}"')
-    def upload_from_fs(self, dirname) -> Bundle:
-        return self._upload(stream.file(dirname))
+    def upload_from_fs(self, dirname, **args) -> BundleList:
+        streams = stream.file(dirname, **args)
```

Comment:
You could change signature in that way:

```
def upload_from_fs(self, **dirnames) -> BundleList:
```

Reply:
dirname is required parameter. **args - named parameter of func build from packer to changes its behavior.